src/utils.rs

```diff
@@ -410,8 +410,6 @@ pub(crate) async fn launch(
     };
     println!("{} was selected", exe.display());
 
-    #[cfg(not(target_os = "windows"))]
-    let should_use_wine = (os == &BuildOs::Windows) && !no_wine;
```

Comment:
I'd prefer if you just set should_use_wine to false on Windows, and keep the current logic of should_use_wine for the other targets. Since the compiler ignores cfgs that don't match, you can have separate implementations for variable initialization per platform

Reply:
The should_use_wine is not really needed, I just use no_wine now.